app/css/bootstrap.css

```diff
@@ -3277,8 +3275,8 @@ fieldset[disabled] .btn-info:focus,
 .btn-info.disabled.focus,
 .btn-info[disabled].focus,
 fieldset[disabled] .btn-info.focus {
-  background-color: #5bc0de;
-  border-color: #46b8da;
+  background-color: #743dfb;
+
```

Comment:
check if this change is making any difference - if not, get it back to how it was

Reply:
all the colors have been changed back to previous